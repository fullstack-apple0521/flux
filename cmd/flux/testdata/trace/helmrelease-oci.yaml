---
apiVersion: v1
kind: Namespace
metadata:
  name: {{ .fluxns }}
---
apiVersion: v1
kind: Namespace
metadata:
  name: {{ .ns }}
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: infrastructure
    kustomize.toolkit.fluxcd.io/namespace: {{ .fluxns }}
  name: podinfo
  namespace: {{ .ns }}
spec:
  chart:
    spec:
      chart: podinfo
      sourceRef:
        kind: HelmRepository
        name: podinfo
        namespace: {{ .fluxns }}
  interval: 5m
status:
  conditions:
  - lastTransitionTime: "2021-07-16T15:42:20Z"
    message: Release reconciliation succeeded
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  helmChart: {{ .fluxns }}/podinfo-podinfo
  lastAppliedRevision: 6.0.0
  lastAttemptedRevision: 6.0.0
  lastAttemptedValuesChecksum: c31db75d05b7515eba2eef47bd71038c74b2e531
---
apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
kind: Kustomization
metadata:
  name: infrastructure
  namespace: {{ .fluxns }}
spec:
  path: ./infrastructure
  sourceRef:
    kind: OCIRepository
    name: flux-system
  validation: client
  interval: 5m
  prune: false
status:
  conditions:
  - lastTransitionTime: "2021-08-01T04:52:56Z"
    message: 'Applied revision: main/696f056df216eea4f9401adbee0ff744d4df390f'
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  lastAppliedRevision: main/696f056df216eea4f9401adbee0ff744d4df390f
---
apiVersion: source.toolkit.fluxcd.io/v1beta2
kind: OCIRepository
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: flux-system
    kustomize.toolkit.fluxcd.io/namespace: {{ .fluxns }}
  name: flux-system
  namespace: {{ .fluxns }}
spec:
  interval: 10m0s
  provider: generic
  ref:
    tag: 1.2.3
  timeout: 60s
  url: oci://ghcr.io/example/repo
status:
  artifact:
    lastUpdateTime: "2022-08-10T10:07:59Z"
    metadata:
      org.opencontainers.image.revision: 6.1.6/450796ddb2ab6724ee1cc32a4be56da032d1cca0
      org.opencontainers.image.source: https://github.com/stefanprodan/podinfo.git
    path: "example"
    revision: dbdb109711ffb3be77504d2670dbe13c24dd63d8d7f1fb489d350e5bfe930dd3
    url: "example"
  conditions:
  - lastTransitionTime: "2021-07-20T00:48:16Z"
    message: "stored artifact for digest 'dbdb109711ffb3be77504d2670dbe13c24dd63d8d7f1fb489d350e5bfe930dd3'"
    reason: Succeed
    status: "True"
    type: Ready
