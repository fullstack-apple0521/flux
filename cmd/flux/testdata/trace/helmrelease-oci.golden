
Object:          HelmRelease/podinfo
Namespace:       {{ .ns }}
Status:          Managed by Flux
---
Kustomization:   infrastructure
Namespace:       {{ .fluxns }}
Path:            ./infrastructure
Revision:        main/696f056df216eea4f9401adbee0ff744d4df390f
Status:          Last reconciled at {{ .kustomizationLastReconcile }}
Message:         Applied revision: main/696f056df216eea4f9401adbee0ff744d4df390f
---
OCIRepository:   flux-system
Namespace:       {{ .fluxns }}
URL:             oci://ghcr.io/example/repo
Tag:             1.2.3
Revision:        dbdb109711ffb3be77504d2670dbe13c24dd63d8d7f1fb489d350e5bfe930dd3
Origin Revision: 6.1.6/450796ddb2ab6724ee1cc32a4be56da032d1cca0
Origin Source:   https://github.com/stefanprodan/podinfo.git
Status:          Last reconciled at {{ .ociRepositoryLastReconcile }}
Message:         stored artifact for digest 'dbdb109711ffb3be77504d2670dbe13c24dd63d8d7f1fb489d350e5bfe930dd3'
